### PR TITLE
[ceph] only check mon elections min 1 hour after boot

### PR DESCRIPTION
**Pull request by dosaboy**
We expect some re-elections are a boot/reboot especially when a cluster is first deployed so ignore the first hour after a boot.

Resolves: #433

### CODE REVIEW

File: hotsos/core/ycheck/engine/properties/search.py [hunk 1]
```diff
@@ -43,11 +43,18 @@ def search_period_hours(self):
...
-        time (in the case of a sosreport would time sosreport was created).
+        Result must have occurred within search-result-age-hours from current
+        time (in the case of a sosreport would be time sosreport was created).
         """
         return int(self.content.get('search-result-age-hours', 0))
 
+    @cached_yproperty_attr
+    def min_hours_since_last_boot(self):
+        """
+        Result must have occurred within hours from last boot.
```
Comment (pponnuvel) [inline hotsos/core/ycheck/engine/properties/search.py:54]: You mean: "[..] *after* hours [..]", right?

Maybe reword it to: "Result must have occurred after this many hours since last boot."?

**Review by pponnuvel — APPROVED** [review]
LGTM

File: hotsos/core/ycheck/engine/properties/search.py (hunk 1 above)
Comment (dosaboy) [reply hotsos/core/ycheck/engine/properties/search.py:54]: yes sorry my docstring says the opposite of what i meant :/ ill fix that now
